src/components/Metatags/Metatags.tsx

```diff
@@ -0,0 +1,187 @@
+
+      {doc.page && <meta name="citation_firstpage" content={doc.page} />}
+
+      {doc.doi && doc.doi.length > 0 && <meta name="citation_doi" content={doc.doi[0]} />}
+
+      {doc.issn && doc.issn.length > 0 && <meta name="citation_issn" content={doc.issn[0]} />}
+
+      {doc.isbn && doc.isbn.length > 0 && <meta name="citation_isbn" content={doc.isbn[0]} />}
+
+      <meta name="citation_language" content="en" />
+
+      {doc.keyword && <meta name="citation_keywords" content={doc.keyword.join('; ')} />}
+
+      {doc.doctype === 'PhD Thesis' && <meta name="citation_dissertation_name" content="Phd" />}
+
+      {doc.doctype === 'Masters Thesis' && <meta name="citation_dissertation_name" content="MS" />}
+
+      <meta name="citation_abstract_html_url" content={url} />
+
+      {doc.pubdate && <meta name="citation_publication_date" content={formatted_numeric_pubdate} />}
+
+      {doc.aff && doc.aff.map((a) => <meta key={a} name="citation_author_institution" content={a} />)}
+
+      {doc.esources && doc.esources.find((e) => e === Esources.PUB_PDF) && (
+        <meta name="citation_pdf_url" content={`${LINKGWAY_BASE_URL}/${doc.bibcode}/PUB_PDF`} />
+      )}
+
+      {doc.page_range && <meta name="citation_lastpage" content={last_page} />}
+
+      {arXiv !== '' && <meta name="citation_arxiv_id" content={arXiv} />}
+
+      <link title="schema(PRISM)" rel="schema.prism" href="http://prismstandard.org/namespaces/1.2/basic/" />
+
+      {doc.pubdate && <meta name="prism.publicationDate" content={formatted_numeric_pubdate} />}
+
+      {doc.bibstem && doc.bibstem.length > 0 && <meta name="prism.publicationName" content={doc.bibstem[0]} />}
+
+      {doc.issn && doc.issn.length > 0 && <meta name="prism.issn" content={doc.issn[0]} />}
+
+      {doc.volume && <meta name="prism.volume" content={doc.volume} />}
+
+      {doc.page && <meta name="prism.startingPage" content={doc.page} />}
+
+      {doc.page_range && <meta name="prism.endingPage" content={last_page} />}
+
+      <link title="schema(DC)" rel="schema.dc" href="http://purl.org/dc/elements/1.1/" />
+
+      {doc.doi && doc.doi.length > 0 && <meta name="dc.identifier" content={`doi:${doc.doi[0]}`} />}
+
+      {doc.pubdate && <meta name="dc.date" content={formatted_numeric_pubdate} />}
+
+      {doc.bibstem && doc.bibstem.length > 0 && <meta name="dc.source" content={doc.bibstem[0]} />}
+
+      <meta name="dc.title" content={title} />
+
+      {doc.author && doc.author.map((a, i) => <meta key={`dcc-${i}`} name="dc.creator" content={a} />)}
+
+      <meta name="twitter:card" content="summary_large_image" />
+
+      <meta name="twitter:description" content={doc.abstract} />
+
+      <meta name="twitter:title" content={title} />
+
+      <meta name="twitter:site" content="@adsabs" />
+
+      <meta name="twitter:domain" content="NASA/ADS" />
+
+      <meta name="twitter:image:src" content={logo} />
+
+      <meta name="twitter:creator" content="@adsabs" />
```

Comment:
I think these are fine for now, this is taken basically straight from how bumblebee is doing things.  
However I think we can modularize this a little bit in a future refactor so we aren't mixing so much logic into the jsx. 
I'm sure we can at least cut down on some of the redundancy anyway